projects/1.20/assets/roost-re-hatched/roost/lang/zh_cn.json

```
@@ -0,0 +1,81 @@
+{
+  "advancements.almost_flying.descr": "将一只栖鸟的最大滑翔距离提升至上限",
+  "advancements.almost_flying.title": "仿若翱翔",
+  "advancements.commanding.descr": "制作命令哨，管理栖鸟的实用工具",
+  "advancements.commanding.title": "听令！",
+  "advancements.delicious_part_2_electric_boogalo.descr": "一种对栖鸟而言更美味的小食",
+  "advancements.delicious_part_2_electric_boogalo.title": "美味的（第二部分）： 霹雳舞动",
+  "advancements.delicious.descr": "制作一份幼虫沙拉，适用于任何鸟的美味小吃",
+  "advancements.delicious.title": "美味的？",
+  "advancements.farming_grubs.descr": "放置你的首个幼虫巢",
+  "advancements.farming_grubs.title": "养殖幼虫",
+  "advancements.fastest_roost_in_the_west.descr": "将一只栖鸟的最大速度提升至上限",
+  "advancements.fastest_roost_in_the_west.title": "西部快栖者",
+  "advancements.flippin_stones.descr": "掀开一块扁石",
+  "advancements.flippin_stones.title": "掀开扁石",
+  "advancements.gettin_grubby.descr": "得到一只幼虫，它们通常藏在扁石下面",
+  "advancements.gettin_grubby.title": "肮脏的收获",
+  "advancements.heavy_duty_poultry.descr": "将一只栖鸟的最大血量提升至上限",
+  "advancements.heavy_duty_poultry.title": "重禽",
+  "advancements.inovations_in_combat.descr": "获得灾厄长柄刀",
+  "advancements.inovations_in_combat.title": "战斗创新",
+  "advancements.mirror_mirror.descr": "制作栖鸟魔镜",
+  "advancements.mirror_mirror.title": "魔镜啊魔镜",
+  "advancements.roost_roost.descr": "让你的栖鸟栖入窝中",
+  "advancements.roost_roost.title": "栖鸟如栖",
+  "advancements.roosting.descr": "哇哦，看看这些鸟",
+  "advancements.roosting.title": "栖鸟",
+  "advancements.shiny_shiny_stones.descr": "寻得你的首颗蛋白石",
+  "advancements.shiny_shiny_stones.title": "闪闪亮石",
+  "advancements.woah_its_blue.descr": "获得一颗栖鸟蛋",
+  "advancements.woah_its_blue.title": "哇！它是蓝色的！",
+  "block.roost.block_of_opal": "蛋白石块",
+  "block.roost.flatstoneblock": "扁石",
+  "block.roost.grub_nest": "幼虫巢",
+  "block.roost.opal_ore": "蛋白石矿石",
+  "block.roost.opaline_glass": "乳色玻璃",
+  "block.roost.opaline_glass_pane": "乳色玻璃板",
+  "block.roost.roost_egg_1": "栖鸟蛋",
+  "block.roost.roost_nest": "栖鸟窝",
+  "enchantment.roost.riders_grace": "骑士优雅",
+  "entity.roost.regal_roost": "栖鸟",
+  "entity.roost.roost": "栖鸟",
+  "entity.roost.roost_rider": "栖鸟骑士",
+  "entity.roost.roostbaby": "栖鸟雏",
+  "item_group.roost.roost_tab": "栖鸟",
+  "item.roost.airy_roost_treat": "轻盈栖鸟款待",
```

```suggestion
  "item.roost.airy_roost_treat": "轻盈栖鸟饲料",
```
这里的airy也许也指蓬松？
看游戏内容